it/google-cloud-platform/src/test/java/com/google/cloud/teleport/it/gcp/bigtable/BigTableIOLT.java

```diff
@@ -0,0 +1,251 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.gcp.bigtable;
+
+import static com.google.cloud.teleport.it.gcp.bigtable.BigtableResourceManagerUtils.generateTableId;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import com.google.auto.value.AutoValue;
+import com.google.bigtable.v2.Mutation;
+import com.google.cloud.bigtable.config.BigtableOptions;
+import com.google.cloud.teleport.it.common.PipelineLauncher;
+import com.google.cloud.teleport.it.common.PipelineOperator;
+import com.google.cloud.teleport.it.common.TestProperties;
+import com.google.cloud.teleport.it.common.utils.ResourceManagerUtils;
+import com.google.cloud.teleport.it.gcp.IOLoadTestBase;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Map;
+import org.apache.beam.sdk.io.GenerateSequence;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.apache.beam.sdk.testing.TestPipelineOptions;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * BigTableIO performance tests.
+ *
+ * <p>Example trigger command for all tests: "mvn test -pl it/google-cloud-platform -am
+ * -Dtest=BigTableIOLT \ -Dproject=[gcpProject] -DartifactBucket=[temp bucket]
+ * -DfailIfNoTests=false".
+ *
+ * <p>Example trigger command for specific test: "mvn test -pl it/google-cloud-platform -am \
+ * -Dtest="BigTableIOLT#testAvroFileLoadsWriteThenRead" -Dconfiguration=local -Dproject=[gcpProject]
+ * \ -DartifactBucket=[temp bucket] -DfailIfNoTests=false".
+ */
+public class BigTableIOLT extends IOLoadTestBase {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BigTableIOLT.class);
+  private static final String COLUMN_FAMILY_NAME = "cf";
+  private static final long TABLE_MAX_AGE_MINUTES = 100L;
+
+  private static BigtableResourceManager resourceManager;
+  private static final String READ_ELEMENT_METRIC_NAME = "read_count";
+  private Configuration configuration;
+  private BigtableOptions bigtableOptions;
+  private String tableId;
+
+  @Rule public TestPipeline writePipeline = TestPipeline.create();
+  @Rule public TestPipeline readPipeline = TestPipeline.create();
+
+  @Before
+  public void setup() throws IOException {
+    resourceManager = DefaultBigtableResourceManager.builder(testName, project).build();
```

Comment:
resource manager should be setup in `@beforeClass`, as it's teardown'd in `@afterClass`

Reply:
Moved teardown to `@After` according to other tests